app_secrets.rb

```diff
@@ -106,6 +107,7 @@ def vault_file(secret_name, file_path, refresh: true)
     mandatory :OIDC_SECRET_KEY, :string
     mandatory :STAGING_OAUTH_CLIENT, :string
     mandatory :STAGING_OAUTH_SECRET, :string
+    mandatory :CURRENCY_LAYER_API_KEY, :string
```

Comment:
Just curious, why is this `mandatory` when we fall back to EuCentralBank outside of prod anyways?

Reply:
You are right, it's not needed. I removed itnow.

Reply:
...that means you probably also don't need the `.env` sample values anymore, or do you?

Reply:
Okay I thought env should be added there, my bad - it's my first encounter with env stuff. I've removed them now.

Reply:
You need `.env.development` entries, if you actually read `AppSecrets.WHATEVER` in development. But in this case, `CURRENCY_LAYER_API_KEY` is only ever read in Prod, and never accessed in Dev or Test. So you don't need to fill values in for `.env.test` nor for `.env.development`.

Reply:
Got it, thanks for the info.